packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts

```diff
@@ -34,22 +34,36 @@ import { isDemoMode } from '../../modes/demo-mode';
 import { isSFDXContainerMode } from '../../util';
 import { ConfigSource, OrgAuthInfo } from '../../util/index';
 import { ForceAuthDemoModeExecutor } from './forceAuthWebLogin';
+import { ForceAuthWebLoginContainerExecutor } from './forceAuthWebLogin';
+
+export class ForceAuthDevHubContainerExecutor extends ForceAuthWebLoginContainerExecutor {
+  public build(data: {}): Command {
+    const command = new SfdxCommandBuilder().withDescription(
+      nls.localize('force_auth_web_login_authorize_dev_hub_text')
+    );
+
+    command
+      .withArg('force:auth:device:login')
```

Comment:
nit: We've (well me) has been pushing to starting using constants for these kind of string values.  Makes testing easier and reduces typo potential.  We don't quite have a good land place for the string yet but if you could just pull them up as constants in the file that'd be great. (note a nit from me means you can decide).  I realize we have this all over currently, but trying to make improvements as we go.

Reply:
Sounds good, please see the [chore: use constants for cli commands](https://github.com/forcedotcom/salesforcedx-vscode/pull/4077/commits/99dab5ccbc87c730be6a76d17ed27bf03eb3913a) commit.